src/components/Comments/WriteComment.js

```diff
@@ -189,7 +190,7 @@ export const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'column',
     // alignItems: 'center',
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: colors.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been updated. Ensure this color change aligns with your design guidelines.
```diff
- backgroundColor: COLORS.lightgrey,
+ backgroundColor: colors.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->